src/config/dynamic-theme-fixes.config

```diff
@@ -2092,29 +2115,6 @@ body, .BA-topbg {
 
 ================================
 
-www.google.*
```

Comment:
I think that `www.` should be still there for Google Search, since there are absolute different products hosted on other subdomains.

Reply:
Why add `www.` If everything works fine even without them?

Reply:
It's because google.com can have many subdomains, and if they are not listed in config, then `google.*` fixes will be applied to them. But we don't know for sure if they are not breaking something on these subdomains.

Also I'm going to make more consistent behavior, treat `google.com` and `*.google.com` as different websites.

Reply:
OK